components/browser/state/src/main/java/mozilla/components/browser/state/action/BrowserAction.kt

```diff
@@ -256,6 +256,15 @@ sealed class LastAccessAction : BrowserAction() {
         val tabId: String,
         val lastMediaAccess: Long = System.currentTimeMillis()
```

Comment:
Do you want to change this now and the docs "use 0 to reset" having the new `ResetLastMediaAccessAction`?

Reply:
Definitely. Thank you!